src/Exceptions.test.ts

```diff
@@ -0,0 +1,56 @@
+import nock from "nock";
+import LensPlatformClient from "./LensPlatformClient";
+import { minimumOptions as options, apiEndpointAddress } from "./LensPlatformClient.test";
+import { NotFoundException, LensSDKException } from "./exceptions";
+
+export const plaform404response = {
+  name: "HTTPError",
+  message: "Response code 404 (Not Found)"
+};
+
+export const plaform500response = {
+  name: "HTTPError",
+  message: "Response code 500 (Internal Server Error)"
+};
+
+describe(".users.*", () => {
+  const invalidUserName = "idonotexist";
+  const brokenUserName = "ibreakthings";
+  const lensPlatformClient = new LensPlatformClient(options);
+
+  beforeAll(() => {
+    nock(apiEndpointAddress).get(`/users/${invalidUserName}`).reply(404, plaform404response);
+    nock(apiEndpointAddress).get(`/users/${brokenUserName}`).reply(500, plaform500response).persist();
+    nock(apiEndpointAddress).get("/users/test").reply(200, { });
+  });
+
+  it("doesn't break functionality", async () => {
+    const result = await lensPlatformClient.user.getOne({ username: "test" });
+
+    expect(result).toStrictEqual({});
```

Comment:
[nit] Minor refactoring to make the test more robust
```suggestion
  const username = "a_random_user"
  const getOneResponse = { username };
  beforeAll(() => {
    nock(apiEndpointAddress).get(`/users/${invalidUserName}`).reply(404, plaform404response);
    nock(apiEndpointAddress).get(`/users/${brokenUserName}`).reply(500, plaform500response).persist();
    nock(apiEndpointAddress).get(`/users/${username}`).reply(200, getOneResponse);
  });

  it("doesn't break functionality", async () => {
    const result = await lensPlatformClient.user.getOne({ username });

    expect(result).toStrictEqual(getOneResponse);
```